invadersgame.py

```diff
@@ -40,7 +40,13 @@ def initialize(self):
 		
 		self.controllers = [self.swarm_controller, self.player_controller, collision_controller, explosion_controller]
 		
-		
+	
+	def reset(self):
+		self.swarmSpeed = 500
```

Comment:
Let me know if anything else needs to be reset at the beginning of a game.  I just added the obvious one I saw because I assume swarm speed increases on level up and I don't see where it gets reset.